test/k8sT/Policies.go

```diff
@@ -1673,6 +1673,22 @@ var _ = Describe("K8sPolicyTest", func() {
 					By("Checking policy correctness")
 					validateNodeConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow)
 				})
+
+				It("Validates fromEntities cluster policy", func() {
+					By("Installing fromEntities cluster policy")
+					importPolicy(kubectl, testNamespace, cnpFromEntitiesCluster, "from-entities-cluster")
+
+					By("Checking policy correctness")
+					validateNodeConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow)
+				})
+
+				It("Validates fromEntities all policy", func() {
+					By("Installing fromEntities cluster policy")
+					importPolicy(kubectl, testNamespace, cnpFromEntitiesAll, "from-entities-all")
+
+					By("Checking policy correctness")
+					validateNodeConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow)
+				})
```

Comment:
I think this should be outside of the `with remote-node identity enabled` Context because it doesn't depend on `enable-remote-node-identity`.

Reply:
Thanks for your comment!
IMO, to make sure that `fromEntities: all` policy really doesn't depend on  `enable-remote-node-identity`, I think we should add this test to `with remote-node identity enabled` and `with remote-node identity disabled`.
However, I don't know the detailed behavior of `enable-remote-node-identity`, so I can't be sure that my opinion is correct.
What do you think?

Reply:
> However, I don't know the detailed behavior of `enable-remote-node-identity`, so I can't be sure that my opinion is correct.

If `enable-remote-node-identity=false`, the `remote-node` entity doesn't exist and the `host` entity includes both the local and remote nodes. If `enable-remote-node-identity=true`, then `remote-node` exists and `host` includes only the local node.

> IMO, to make sure that `fromEntities: all` policy really doesn't depend on  `enable-remote-node-identity`, I think we should add this test to `with remote-node identity enabled` and `with remote-node identity disabled`.

That's a good point. However, end-to-end tests are costly, so I don't think it's worth the extra tests. We already have more fine-grained unit tests to validate those things.

Reply:
Thanks!
Yes, I agree with you. I'll put this test outside of that context.